server/.golangci.yml

```diff
@@ -134,7 +134,7 @@ issues:
         channels/app/post.go|\
         channels/app/post_helpers_test.go|\
         channels/app/post_metadata.go|\
-        channels/app/post_persistent_notification_test.go|\
+        channels/app/post_metadata_test.go|\
```

Comment:
The diff looks off. Can you please fix that? Thanks!

Reply:
I guess it looks good! 😑

Reply:
Done!